docs/data/joy/getting-started/tutorial/LoginFinal.js

```diff
@@ -42,7 +38,7 @@ export default function App() {
         <ModeToggle />
         <Sheet
           sx={{
-            maxWidth: 400,
+            width: 300,
```

Comment:
So it looks better?

**Before**
<img width="712" alt="Screenshot 2022-10-07 at 17 39 37" src="https://user-images.githubusercontent.com/3165635/194593512-7fe2fe03-b44e-479d-bbfd-b0ae90147a6d.png">

**After**
<img width="725" alt="Screenshot 2022-10-07 at 17 39 29" src="https://user-images.githubusercontent.com/3165635/194593531-1bdc72c4-1598-4e85-b3bd-0420e286c323.png">




**After**